README.md

```diff
@@ -1,10 +1,40 @@
-sonar-grammar 
+Sonar-grammar
 =============
 
 Sonar Grammar Plugin to provide spell checking of source code. 
 
 As a spell checking library Jazzy is used http://jazzy.sourceforge.net/
 
+Requirements environment:
+==========================
+1. Java 1.7
+2. Maven
+3. SonarQube(current version 5.1)
+
+SonarQube install steps:
+========================
+1. Download SonarQube (current version 5.1) from  [Sonarqube homepage](http://www.sonarqube.org/downloads/)
+2. Unpack and go to \sonarqube-5.1\bin\ folder
+3. Select you current OS and run sonar(can be ".bat" or ".sh" file)
+4. In browser go to -  [http://localhost:9000/](http://localhost:9000/)
+5. Login in Sonar as Administrator (login - admin, password - admin)
+6. On Sonar page top menu go to the "Quality Profiles". You need to make sure that "Sonar way " stands as the default
+7. Open terminal(comand line), go to the project that you want to test and write "mvn sonar:sonar" command. For debug use "mvnDebug sonar:sonar" command(port 8000)
+8. After that you can see results on [http://localhost:9000/dashboard/](http://localhost:9000/dashboard/) or follow the link that will be found in bottom command line
+
+Sonar-Grammar-Plugin use:
+=========================
+1. Use maven for build project
+2. In "sonar-grammar\target\" folder will be "sonar-grammar-plugin-XXX.jar" file. Copy this file to local SonarQubefolder - "sonarqube-5.1\extensions\plugins\"
```

Comment:
please check this Maven plugin for dev purposes https://github.com/SonarSource/sonar-dev-maven-plugin